index.html

```diff
@@ -4,17 +4,21 @@
     <meta charset="UTF-8" />
     <meta http-equiv="X-UA-Compatible" content="IE=edge" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="style.css">
     <title>Library Book App</title>
   </head>
   <body>
-    <h1 class="text-4xl">Awesome Books</h1>
-    <div class="books-container" id="bookslist">
-        <p id="bookstatus"></p>
-    </div>
-    <div class="book-form-container">
-      <input type="text" placeholder="Title" id="title"/><br><br>
-      <input type="text" placeholder="Author" id="author"/><br><br>
-      <button id="addbook">Add</button>
+    <div class="container">
+      <h1 class="logo">Awesome Books</h1>
+      <div class="books-container" id="bookslist">
+          <p id="bookstatus"></p>
+      </div>
+      <div class="book-form-container">
+        <h3 class="subtitle">Add a new book</h3>
+        <input class="input" type="text" placeholder="Title" id="title"/><br><br>
+        <input class="input" type="text" placeholder="Author" id="author"/><br><br>
+        <button type="button" id="addbook">Add</button>
```

Comment:
Please follow HTML semantic rule here and wrap input fields and buttons with the `form` tag.